### ISSUE TITLE
Strokes are not closed in desktop

### ISSUE THREAD

**Issue by krazzbeluh** [korlibs/korge#373]
Hi,
I created a roundRectand a uiButton with strokes.
The problem is when I compile to desktop application (I only tested on Windows), this stroke is not closed at the top-right corner.

![image](https://user-images.githubusercontent.com/21320222/122828872-8031f200-d2e6-11eb-900b-c03e871260c7.png)
![image](https://user-images.githubusercontent.com/21320222/122828891-8627d300-d2e6-11eb-8a88-3265b4718514.png)

Thanks

**Comment by soywiz** [comment]
Yeah, unfortunately software rasterizer still have some issues like that one. I have to check it 👀

**Comment by soywiz** [comment]
I guess this happens on windows and linux. But I guess macOS has a native vector implementation. For windows (in addition to fixing software rasterizer): https://github.com/korlibs/korge-next/issues/81

**Comment by krazzbeluh** [comment]
I tested in macOS and it happens too but a bit less obvious

<img width="120" alt="Capture d’écran 2021-06-22 à 16 19 36" src="https://user-images.githubusercontent.com/21320222/122941404-a783d000-d375-11eb-9627-f79a061c8133.png">
<img width="286" alt="Capture d’écran 2021-06-22 à 16 16 57" src="https://user-images.githubusercontent.com/21320222/122941435-af437480-d375-11eb-82eb-2994727186d5.png">
